bees/execbee/execbee.go

```diff
@@ -45,13 +47,31 @@ func (mod *ExecBee) Action(action bees.Action) []bees.Placeholder {
 	switch action.Name {
 	case "execute":
 		var command string
+		var stdin string
+
 		action.Options.Bind("command", &command)
+		action.Options.Bind("stdin", &stdin)
 		mod.Logln("Executing locally: ", command)
 
 		go func() {
 			c := strings.Split(command, " ")
 			cmd := exec.Command(c[0], c[1:]...)
 
+			if stdin != "" {
+				stdinPipe, err := cmd.StdinPipe()
+
+				if err != nil {
+					mod.LogFatal("Error creating stdinPipe for Cmd", err)
+					return
+				}
+
+				go func() {
```

Comment:
Just curious: is there a reason you execute this in the background?

Reply:
No, I just took that from the go-exec documentation. But you are right, it it not necessary to run this in background.

Reply:
Ok, that got me intrigued. There must be a reason they use a separate goroutine in the example... and turns out there is:

The command isn't actually being started until you call CombinedOutput(). Writing to the StdinPipe would block hence.

So: your code is fine, ignore my remark!